mlir/test/Catalyst/AsyncException.mlir

```diff
@@ -216,6 +216,6 @@ module {
   ^bbbad:
      // CHECK: llvm.call @__catalyst__host__rt__unrecoverable_error
      llvm.call @abort() : () -> ()
-     llvm.br ^finally
+     llvm.unreachable
```

Comment:
This change affects the logic of the program and must be placed in a different commit that is not documentation